Makefile

```diff
@@ -96,6 +96,9 @@ lint: $(GOLANGCILINT)
 validate-swagger:
 	swagger generate spec --scan-models -o ./swagger.yaml && swagger validate ./swagger.yaml
 
+serve-swagger:
+	swagger generate spec --scan-models -o ./swagger.yaml && swagger validate ./swagger.yaml && swagger serve ./swagger.yaml --no-open
```

Comment:
Coudn't it be:
```suggestion
serve-swagger: validate-swagger
	swagger serve ./swagger.yaml --no-open
```

Reply:
fixed. I didn't now we can do this, it's a lot better!